app/src/layouts/appLayout/footer/footer.jsx

```diff
@@ -0,0 +1,83 @@
+import React, { PureComponent } from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import { FormattedMessage } from 'react-intl';
+import { buildVersionSelector } from 'controllers/appInfo';
+import { referenceDictionary } from 'common/utils';
+import styles from './footer.scss';
+
+const cx = classNames.bind(styles);
+
+@connect(state => ({
+  buildVersion: buildVersionSelector(state),
+}))
+export class Footer extends PureComponent {
```

Comment:
Connected components are pure by default, so no need to use `PureComponent` here

Reply:
done